src/pykeen/pipeline/api.py

```diff
@@ -524,8 +524,8 @@ def replicate_pipeline_from_config(
 
 def _iterate_moved(pipeline_results: Iterable[PipelineResult]):
     for pipeline_result in pipeline_results:
-        pipeline_result.model.device = resolve_device("cpu")
-        pipeline_result.model.to_device_()
+        pipeline_result.model = pipeline_result.model.cpu()
+        torch.cuda.empty_cache()
```

Comment:
is the empty cache needed?

Reply:
I do not know :sweat_smile: 

I am always wondering why we manually need to clear caches, but I remember that at some point we got errors when not doing so. @lvermue might remember when this happened and maybe why :wink: